src/components/confirmation/README.md

```diff
@@ -0,0 +1,50 @@
+# Confirmation
+
+The `Confirmation` component is used to wrap any custom notification component.
```

Comment:
Hmm, what about: `is used in case of components where you need an additional confirmation step before performing a dangerous and permanent action like removing an item from the database.`